avogadro/qtplugins/crystal/crystal.cpp

```diff
@@ -152,11 +160,24 @@ void Crystal::updateActions()
     foreach (QAction *action, m_actions)
       action->setEnabled(false);
 
+    m_importCrystalClipboardAction->setEnabled(true);
     m_toggleUnitCellAction->setEnabled(true);
     m_toggleUnitCellAction->setText(tr("Add &Unit Cell"));
   }
 }
 
+void Crystal::importCrystalClipboard()
+{
+  ImportCrystalDialog d;
+  Avogadro::Core::Molecule m;
```

Comment:
You don't need the Avogadro:: qualifier as you are already in the Avogadro namespace